Gruntfile.js

```diff
@@ -86,89 +96,6 @@ module.exports = function(grunt) {
       }
     }
   });
-
-  grunt.task.registerMultiTask(
```

Comment:
The point of these two grunt tasks is to take the transmuxer_worker.js file, inline it's contents (which is admittedly done by browserify now so that is ok) and then create a blob worker and change the `new Worker` line in videojs-media-sources.js so that instead of pointing to a file, it points to an inlined web worker created via a blob object. This allows us to use a web worker without making an extra request to the server for a file (in this case transmuxer_worker.js.)

I'm not sure current build outputs a truly "stand-alone" file with worker without doing at least some of the gymnastics that we previously performed!